frappe/database/database.py

```diff
@@ -55,7 +55,7 @@ class Database:
 	STANDARD_VARCHAR_COLUMNS = ("name", "owner", "modified_by")
 	DEFAULT_COLUMNS = ["name", "creation", "modified", "modified_by", "owner", "docstatus", "idx"]
 	CHILD_TABLE_COLUMNS = ("parent", "parenttype", "parentfield")
-	MAX_WRITES_PER_TRANSACTION = 200_000
```

Comment:
hey, this isn't a bug. We intentionally bumped this number. 

You can fix he docstring if required.

Reply:
ok, that's exactly what I wanted to confirm! thanks.

Reply:
updated the docstring.